docs/pet-kata/slides.md

```diff
@@ -909,26 +909,26 @@ public void getAgeStatisticsOfPets()
 {
   // Try to use a MutableIntList here instead
   // Hints: flatMap = flatCollect, map = collect, mapToInt = collectInt
-  MutableIntList petAges = this.people
-      .flatCollect(Person::getPets)
-      .collectInt(Pet::getAge);
+  var petAges = this.people
+          .flatCollect(Person::getPets)
+          .collectInt(Pet::getAge);
 
   // Try to use an IntSet here instead
   IntSet uniqueAges = petAges.toSet();
 
   // IntSummaryStatistics is a class in JDK 8 - Try and use it with MutableIntList.forEach().
```

Comment:
`// IntSummaryStatistics is a class in JDK 8 - Look at MutableIntList.summaryStatistics().`